python/paddle/fluid/dygraph/jit.py

```diff
@@ -525,6 +528,58 @@ def _build_load_path_and_config(path, config):
     return model_path, config
 
 
+_save_pre_hooks_lock = threading.Lock()
+_save_pre_hooks = []
+
+
+class HookRemoveHelper(object):
+    """ A HookRemoveHelper that can be used to remove hook. """
+
+    def __init__(self, hook):
+        self._hook = hook
+
+    def remove(self):
+        remove_save_pre_hook(self._hook)
+
+
+def register_save_pre_hook(hook):
```

Comment:
we should append api doc like other api

Reply:
Done, added api doc to public functions.